lib/options.js

```diff
@@ -27,19 +30,27 @@ const DEFAULT_OPTIONS = {
     getLogger: defaultLogProvider,
 };
 
+function defaultDatabaseUrl() {
+    const dbUsername = process.env.DATABASE_USERNAME || 'unleash_user';
+    const dbPassword = process.env.DATABASE_PASSWORD || 'password';
+    const dbHost = process.env.DATABASE_HOST || 'localhost';
+    const dbPort = process.env.DATABASE_PORT || 5432;
+    const dbName = process.env.DATABASE_NAME || 'unleash';
+    return `postgres://${dbUsername}:${dbPassword}@${dbHost}:${dbPort}/${dbName}`;
+}
+
 module.exports = {
     createOptions: opts => {
         const options = Object.assign({}, DEFAULT_OPTIONS, opts);
 
         // If we are running in development we should assume local db
         if (isDev() && !options.databaseUrl) {
-            options.databaseUrl =
-                'postgres://unleash_user:passord@localhost:5432/unleash';
+            options.databaseUrl = defaultDatabaseUrl();
```

Comment:
This does not make sense after changes on line 12.

Reply:
This line for ensure we do not change behavior if user didn't spec DATABASE_URL or DATABASE_HOST when run on local develop.